### ISSUE TITLE
Attachement return [File of type 'application/pdf'] insted of URL or icon with NodeJS

### ISSUE THREAD

**Issue by Joffrey4** [Microsoft/BotFramework-Emulator#143]
Hello,

I want by bot to send .pdf files to the user. So, i'm attaching the file to a message with _addAttachment_. But when i run my code in the emulator, it doesn't show any link. There's only a message with **[File of type 'application/pdf']**.

Here is what's happen:
![screenshot at 2017-03-24 12 27 55](https://cloud.githubusercontent.com/assets/17571457/24293562/b092b88c-1092-11e7-82bc-9a78c645f41c.png)

And here is my code:
```
intents.matches(/^recevoir pdf/i, [
    function (session) {
        var msg = new builder.Message(session)
            .addAttachment({
                contentUrl: "https://qconlondon.com/system/files/presentation-slides/microsoft_bot_framework_best_practices.pdf",
                contentType: 'application/pdf',
                name: 'microsoft_bot_framework_best_practices.pdf'
            });

        session.send(msg);
    },
]);
```
My .pdf link is a direct link, and publicy accessible. I've tried to send a picture with _.addAttachement_, and it works. But it doesn't works with any other kind of file.

I'm using BotBuilder 3.7.0 with NodeJS and BotFramework Emulator 3.5.25. I don't understand why it doesn't show me the link of the .pdf. Can you please help me ?

**Comment by eanders-MS** [comment]
Thanks for taking the time to report this. A fix has been made and should be included in the next emulator release.

**Comment by Joffrey4** [comment]
Yes, i saw your fix. Thank you @eanders-MS !

**Comment by eanders-MS** [comment]
Fixed with build 3.4.26
